### PR TITLE
*: Cut v2.9.0 release

### PR DESCRIPTION
**Pull request by rexagod**
## v2.9.0 / 2023-05-23

The changes mentioned below are only the user-facing ones. For a list of the complete set of changes, refer the changelog associated with the release tag.

Note: The experimental VerticalPodAutoscaler metrics are no longer supported, and have been removed.

* [ENHANCEMENT] Add namespace label to `kube_lease_renew_time` #2073 @A-Hilaly
* [FEATURE] Add support for variable VKs in CRS config #1851 @rexagod
* [ENHANCEMENT] Update go version to 1.20.4 #2056 @RamakrishnanArun
* [ENHANCEMENT] Add support for multi cluster prometheus alerts #2058 @jkroepke
* [BUGFIX] Adhere to OTel-Prometheus standard for labels #2004 @rexagod
* [BUGFIX] Support LabelsFromPath functionality for Info-typed metrics #2048 @murphd40
* [ENHANCEMENT] Add parameters for PodSecurity #2042 @jcpunk
* [FEATURE] Use prometheus/exporter-toolkit landing page #2034 @mrueg
* [CHANGE] Remove VerticalPodAutoscaler #2017 @mrueg
* [ENHANCEMENT] Add StatefulSet Start Ordinal metrics for KEP-3335 #1959 @pwschuurman
* [ENHANCEMENT] Only use OpenMetrics and Text in contentType #2024 @CatherineF-dev
* [FEATURE] Support quantities and percentages #1989 @mrueg
* [BUGFIX] Respect relative paths for label resolutions #2007 @rexagod
* [ENHANCEMENT] Don't crash on non-existent path values #1998 @rexagod

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
+## v2.9.0 / 2023-05-23
+
+The changes mentioned below are only the user-facing ones. For a list of the complete set of changes, refer the changelog associated with the release tag.
+
+Note: The experimental VerticalPodAutoscaler metrics are no longer supported, and have been removed.
+
+* [ENHANCEMENT] Add namespace label to `kube_lease_renew_time` #2073 @A-Hilaly
```
[the remark this answers is not in the document]
Comment (rexagod) [reply CHANGELOG.md:7]: ACK

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,24 @@
+## v2.9.0 / 2023-05-23
+
+The changes mentioned below are only the user-facing ones. For a list of the complete set of changes, refer the changelog associated with the release tag.
+
+Note: The experimental VerticalPodAutoscaler metrics are no longer supported, and have been removed.
```
Comment (mrueg) [inline CHANGELOG.md:5]:
```suggestion
Note: The deprecated experimental VerticalPodAutoscaler metrics are no longer supported, and have been removed. We recommend to use CustomResourceState metrics to gather metrics from custom resources like the Vertical Pod Autoscaler. 
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,24 @@
...
+
+* [BUGFIX] Adhere to OTel-Prometheus standard for labels #2004 @rexagod
```
Comment (mrueg) [inline CHANGELOG.md:7]: Was this possibly a breaking change for experimental _label metrics and should be called out as well?

File: README.md [hunk 4]
```diff
@@ -73,11 +73,11 @@ Generally, it is recommended to use the latest release of kube-state-metrics. If
 
 | kube-state-metrics | Kubernetes client-go Version |
 |--------------------|:----------------------------:|
-| **v2.4.2**         | v1.23                        |
 | **v2.5.0**         | v1.24                        |
 | **v2.6.0**         | v1.24                        |
 | **v2.7.0**         | v1.25                        |
 | **v2.8.1**         | v1.26                        |
+| **v2.9.0**         | v1.26                        |
```
Comment (mrueg) [inline README.md:80]: Further below there's `registry.k8s.io/kube-state-metrics/kube-state-metrics:v2.8.1` which should be updated as well (I forgot with the v2.8.2 release)

File: README.md [hunk 5]
```diff
@@ -73,11 +73,11 @@ Generally, it is recommended to use the latest release of kube-state-metrics. If
 
 | kube-state-metrics | Kubernetes client-go Version |
 |--------------------|:----------------------------:|
-| **v2.4.2**         | v1.23                        |
-| **v2.5.0**         | v1.24                        |
 | **v2.6.0**         | v1.24                        |
 | **v2.7.0**         | v1.25                        |
 | **v2.8.1**         | v1.26                        |
```
Comment (mrueg) [inline README.md:78]: Can you remove the v2.8.1 here? We would track it per minor release.

**Comment by mrueg** [comment]
/hold
/lgtm

thanks! Feel free to cancel the hold when you want to proceed.

**Pull request merged**